### DISCUSSION TITLE
Folders view for flows

### DISCUSSION THREAD

**Discussion by rromul** [directus/directus#18843] [Feature Requests]
### Summary

For those projects having numerous flows it would be nice to have folders to group together relative flows

### Motivation

The comfortability in the usage of flows

### Detailed Design

The Flows view can have a switch of 2 views: List/table and Folders. Default view is the actual List view
List view is the actual flows view. Folders view is the tree view

### Must Have:
- Tree view
- List view

### Should Have:
- Views switcher between both

### Drawbacks

No reasons to not to do this

### Alternatives

N/A

### Adoption Strategy

There are no breaking changes

**Comment by emahuni** [comment]
yes, and it would also be good if filtering is enabled for too. If the project has a lot of operations, just finding one is a daunting task. I hope this can be done for operations as well.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
